sockeye/train.py

```diff
@@ -368,7 +350,8 @@ def create_data_iters_and_vocabs(args: argparse.Namespace,
 def create_encoder_config(args: argparse.Namespace,
                           max_seq_len_source: int,
                           max_seq_len_target: int,
-                          config_conv: Optional[encoder.ConvolutionalEmbeddingConfig]) -> Tuple[encoder.EncoderConfig,
+                          config_conv: Optional[encoder.ConvolutionalEmbeddingConfig],
+                          num_embed_source: int) -> Tuple[encoder.EncoderConfig,
                                                                                                 int]:
```

Comment:
indent?

Reply:
yeah the indentation was kind of weird. fixed.